src/NzbDrone.Core/MediaCover/MediaCoverService.cs

```diff
@@ -215,8 +219,11 @@ private void EnsureResizedCovers(Movie movie, MediaCover cover, bool forceResize
         public void Execute(EnsureMediaCoversCommand command)
         {
             var movie = _movieService.GetMovie(command.MovieId);
```

Comment:
Can this entire area be simplified to work like Sonarr meaning get rid of the separate `EnsureMediaCoversCommand`, get rid of the call and pull from `_movieService`. Handle the EnsureCovers logic and the PublishEvent for Updated in the Handle(MovieUpdatedEvent) using the message.movie property already passed in... Also blitz the Handle of MovieAddedEvent that is not doing anything. 

IE. https://github.com/Sonarr/Sonarr/blob/556bd117259eb8ae110a617f79ab87a951b0835d/src/NzbDrone.Core/MediaCover/MediaCoverService.cs#L190